example/src/main/java/io/netty/example/http2/helloworld/server/HelloWorldHttp2HandlerBuilder.java

```diff
@@ -31,6 +31,9 @@
 
     public HelloWorldHttp2HandlerBuilder() {
         frameLogger(logger);
+        // Running h2spec 5.5. that will send a large list without checking our max header list size
+        // so we need to set max header list size to pass the test
+        initialSettings().maxHeaderListSize(Integer.MAX_VALUE);
```

Comment:
@Scottmitch 
It's ok to remove this code after rebase to latest 4.1 branch!

One another question.
Should I not to close the header continuation here, let the api user take care of it?
Or keep the code as is here is fine!
```
    private void verifyUnknownFrame() throws Http2Exception {
        if (headersContinuation != null) {
            int streamId = headersContinuation.getStreamId();
            closeHeadersContinuation();
            throw connectionError(PROTOCOL_ERROR, "Extension frames must not be in the middle of headers "
                    + "on stream %d", streamId);
        }
    }
```